jaeger-thrift/src/test/java/com/uber/jaeger/thrift/reporters/protocols/ThriftSenderBaseTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2016-2018, Uber Technologies, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.uber.jaeger.thrift.reporters.protocols;
+
+import static org.junit.Assert.assertEquals;
+import com.uber.jaeger.agent.thrift.Agent;
+import com.uber.jaeger.thrift.senders.ThriftSenderBase;
+import com.uber.jaeger.thriftjava.Batch;
+import com.uber.jaeger.thriftjava.Process;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.thrift.protocol.TCompactProtocol;
+import org.apache.thrift.transport.AutoExpandingBufferWriteTransport;
+import org.junit.Test;
+
+public class ThriftSenderBaseTest {
+  static final String SERVICE_NAME = "test-sender";
+  final int maxPacketSize = 1000;
+
+  @Test
+  public void testEmitBatchOverhead() throws Exception {
+    int a = calculateBatchOverheadDifference(1);
+    int b = calculateBatchOverheadDifference(2);
+
+    // This value has been empirically observed to be 56.
```

Comment:
I know it's just a code being moved from the old test to the new one, but is this comment accurate? Which value is it referring to? If it's `EMIT_BATCH_OVERHEAD`, then it's 33, not 56...

Reply:
Yeah noticed that too - but wanted to preserve what was already there.
@yurishkuro ok to update the comment, as it looks like it should be 33?